Bangladesh/Demographic_&_Poverty_Data_Merge.py

```diff
@@ -0,0 +1,15 @@
+## Merge the GIS data with the demographic data
+
```

Comment:
I think there are different packages that need to imported for this script to run. From what I can see at least GeoPandas and Pandas. Please add those in.

Reply:
Sure.